htdocs/api/v0.0.1/Login.php

```diff
@@ -129,6 +133,13 @@ function getEncodedToken($user)
                  );
 
         $key = $config->getSetting("JWTKey");
+        if (!isKeyStrong($key)) {
+            error_log(
+                'ERROR: JWTKey config variable is weak. '
+                .'Please change the key to a more cryptographically-secure value.'
+            );
+            return "";
```

Comment:
@xlecours I'm going to leave it here. The empty string is returned to the calling function which will pass a `500` afterward. I think placing the status code in the other function is more readable.